tests/testthat/_snaps/rows-dm.md

```diff
@@ -94,6 +94,9 @@
       dm_copy %>% dm_rows_update(dm_update_copy) %>% pull_tbl(tf_2) %>% arrange_all()
     Message
       Result is returned as a dm object with lazy tables. Use `in_place = FALSE` to mute this message, or `in_place = TRUE` to write to the underlying tables.
+    Condition
+      Warning:
+      `x` must be a one- or two-column data frame in `deframe()`.
```

Comment:
Note to self: Track down where these warnings are coming from and get rid of them.